.github/workflows/release.yml

```
@@ -0,0 +1,328 @@
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.10", "3.11", "3.12", "3.13", "3.14"]
+    steps:
+      - uses: actions/checkout@v7.0.0
+      - name: Set up Python
+        uses: actions/setup-python@v6.3.0
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: TestPyPI install smoke
+        run: python scripts/testpypi-smoke.py ${{ needs.preflight.outputs.version }}
```

actionlint (1.7.12)

[error] property "preflight" is not defined in object type {testpypi-publish: {outputs: {}; result: string}}

(expression)